collections/README.adoc

```diff
@@ -1,3 +1,51 @@
 = Collections good practices
 
-NOTE: Work in Progress...
+Note: Unreviewed work. Please contribute to the discussion in the Automation Red Hat COP
+
+=== Collection Structure
```

Comment:
The rule name should be a one-liner summarizing the rule, so that people can overlook at once the recommendations and only need to uncollapse the ones they need details for.